docs/soundmotion.html

```diff
@@ -397,8 +398,8 @@ <h2>Course Options</h2>
           </li>
         </ul>
       </div>
-    </div>
-    <div class="footer-wrap">
+    </section>
+    <section class="footer-wrap">
```

Comment:
@WillStill Using `<section>` elements is a big improvement for web accessibility. Excellent.